wizard/WizardModeSelection.qml

```diff
@@ -65,52 +66,67 @@ Rectangle {
             WizardMenuItem {
                 opacity: appWindow.persistentSettings.nettype == 0 ? 1.0 : 0.5
                 Layout.topMargin: 20
-                headerText: qsTr("Simple mode") + translationManager.emptyString
+                headerText: qsTr("Simple mode") + (appWindow.persistentSettings.nettype == 0 ? " " + qsTr("(recommended)") : "") + translationManager.emptyString
```

Comment:
I disagree with this change. We should (and we do) recommend users to run their own node, not to rely on a third party one. I would just take off "recommended" from the simple mode.

Reply:
From the screenshots simple mode (bootstrap) gets recommended which means users ending up with their own node.

Reply:
Newbie users usually just click on the reccomended options.
In my design, the user who clicks on both (recommended) options (Simple mode and Local node & Bootstrap node) in the end will have their own local node running.

Reply:
My point is that using bootstrap mode they will transact on the network using a remote node until their local node is synced, which may never happen. I don't know if we should specifically suggest it.